bin.src/certifyCalibration.py

```diff
@@ -67,11 +67,11 @@
     butler = Butler(args.root, run=args.inputCollection)
 
     # Do the thing.
-    crozier = BlessCalibration(butler=butler,
-                               inputCollection=args.inputCollection,
-                               outputCollection=args.outputCollection)
+    certify = CertifyCalibration(butler=butler,
+                                 inputCollection=args.inputCollection,
+                                 outputCollection=args.outputCollection)
 
-    crozier.findInputs(args.datasetTypeName)
+    certify.findInputs(args.datasetTypeName)
     if not args.skipCL:
-        crozier.addCalibrationLabel(beginDate=args.beginDate, endDate=args.endDate)
-    crozier.registerCalibrations(args.datasetTypeName)
+        certify.addCalibrationLabel(beginDate=args.beginDate, endDate=args.endDate)
+    certify.registerCalibrations(args.datasetTypeName)
```

Comment:
`certifier` as a noun perhaps? Don't much care really as it's in a bin script though.